cypress/e2e/tx-builder/tx-builder.spec.cy.js

```diff
@@ -44,7 +44,7 @@ describe('Testing Tx-builder safe app', { defaultCommandTimeout: 12000 }, () =>
     cy.findByRole('region').should('exist')
     cy.findByText('test Address Value').should('exist')
     cy.findByText('newValue(address):').should('exist')
-    cy.findAllByText('0x49d4...26A6').should('have.length', 2)
+    cy.findAllByText('0x49d4...26A6').should('have.length', 1)
```

Comment:
The recent `safe-react-component` implementation in `web-core` broke these tests (see CI on `development`).